CONTRIBUTING.md

```diff
@@ -190,15 +190,40 @@ that can help you.
 
 ## Content contributors
 
-### Creating a news post
+### Creating a blog post
 
-*Coming soon! We are still figuring out exactly how this will look*
+Blog posts must be created in the ``conda-dot-org/blog`` directory with a file naming convention of ``YYYY-MM-DD-title.md``. 
```

Comment:
```suggestion
Blog posts must be created in the `blog` directory with a file naming convention of `YYYY-MM-DD-title.md`. 
```
@kathatherine, I think just saying `blog` directory would be okay. Reading `conda-dot-org/blog` confuses me a little as I'm likely to take this literally and assume there is actually a folder called `conda-dot-org` in this repository.

Also, in Markdown, you don't need to use double backticks. Just one will work.